### PR TITLE
Update Rstudio Workbench

### PR DESCRIPTION
**Pull request by nahara7**
Adds in changes from PR #165 and #162 of the `sol-eng-helm` repo.

### CODE REVIEW

File: charts/rstudio-workbench/README.md.gotmpl [hunk 1]
```diff
@@ -52,8 +19,7 @@ required by RStudio Server Pro. Those config files and their mount locations are
 - The prestart script for RStudio Server is highly customized to:
   - Get the service account information off of the RStudio Server pod for use in launching jobs
   - Generate `launcher.pub` as needed (if `launcher.pem` is provided). If it is not provided,
-  the helm chart will generate it automatically but this information will be lost for subsequent deployments
-  and can cause users to be locked out sessions started by a previous deployment.
+  the helm chart will generate it (which changes each release)
```
Comment (jonyoder) [inline charts/rstudio-workbench/README.md.gotmpl:22]: I think we want the previous text here:
> the helm chart will generate it automatically but this information will be lost for subsequent deployments and can cause users to be locked out sessions started by a previous deployment.

File: charts/rstudio-workbench/README.md.gotmpl [hunk 2]
```diff
@@ -7,39 +7,6 @@
 
 {{ template "rstudio.install" . }}
 
-## Required Configuration
```
Comment (jonyoder) [inline charts/rstudio-workbench/README.md.gotmpl:10]: Was this section accidentally removed?

**Review by jonyoder — CHANGES_REQUESTED** [review]
I noticed a couple of small issues in the README. Other than that, looks great!

**Review by jonyoder — APPROVED** [review]

**Review by kfeinauer — APPROVED** [review]
Awesome stuff!

**Pull request merged**